.github/workflows/packages.yml

```diff
@@ -0,0 +1,130 @@
+name: Build Packages
+
+on:
+  push:
+    branches: [ master ]
+    tags: ['*']
+    paths-ignore: ['**.md']
+  pull_request:
+    branches: [ master ]
+    paths-ignore: ['**.md']
+  workflow_dispatch:
+
+env:
+  VCPKG_BINARY_SOURCES: "clear;x-gha,readwrite"
+
+jobs:
+  build:
+    name: ${{ matrix.config.name }}
+    runs-on: ${{ matrix.config.os }}
+
+    strategy:
+      matrix:
+        config:
+          - name: Linux GCC
+            os: ubuntu-22.04
+            triplet: x64-linux-dynamic-release
+            artifact-name: AppImage
+            artifact-path: build/*.appimage
+
+          - name: MSVC x64
+            os: windows-latest
+            arch: x64
+            triplet: x64-windows-static-release
+            artifact-name: Win-64
+            artifact-path: build/*.zip
+
+          - name: MSVC x86
+            os: windows-latest
+            arch: x86
+            triplet: x86-windows-static-release
+            artifact-name: Win-32
+            artifact-path: build/*.zip
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install dependencies (Linux)
+        if: runner.os == 'Linux'
+        run: |
+          sudo sed -i 's/# deb-src/deb-src/' /etc/apt/sources.list
+          sudo apt-get update
+          sudo apt-get install ninja-build
+          sudo apt-get build-dep libsdl2-dev
+
+      - name: Developer Command Prompt
+        if: runner.os == 'Windows'
+        uses: ilammy/msvc-dev-cmd@v1
+        with:
+          arch: ${{ matrix.config.arch }}
+
+      - name: Export GitHub Actions cache environment variables
+        uses: actions/github-script@v7
+        with:
+          script: |
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+
+      - name: Configure (Linux)
+        if: runner.os == 'Linux'
+        shell: bash
+        run: |
+          cmake -B build -G Ninja -DCMAKE_BUILD_TYPE=Release \
+            -DENABLE_WERROR=ON -DENABLE_LTO=ON \
```

Comment:
No more `-DENABLE_HARDENING=ON` ?

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1724/commits/09988544a54b1db0a9c2dbc022fdc812fdf9a543